### PR TITLE
feat: Improve UI of Tickets Fragment

### LINKED ISSUE #249: Improve UI of tickets Fragment
as discussed in the meeting
working on it 👍

### PR DESCRIPTION
**Pull request by simarsingh24**
fixes #249

### CODE REVIEW

**Comment by simarsingh24** [comment]
SS after the change-
screenshot_20180704-005602

File: app/src/main/res/layout/item_ticket.xml [hunk 1]
```diff
@@ -30,23 +30,27 @@
...
+            android:background="@drawable/filled_border"
+            android:layout_weight="1">
         <TextView
             android:id="@+id/order"
             android:layout_width="0dp"
-            android:layout_height="wrap_content"
+            android:layout_height="30dp"
             android:layout_weight="1"
             android:gravity="end"
-            android:spinnerMode="dialog"
```
Comment (nikit19) [inline app/src/main/res/layout/item_ticket.xml]: why removed?

File: app/src/main/res/layout/item_ticket.xml [hunk 2]
```diff
@@ -30,23 +30,27 @@
             android:layout_weight="1"
             android:textSize="@dimen/text_size_large"
             tools:text="$25" />
-
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:background="@drawable/filled_border"
+            android:layout_weight="1">
```
Comment (nikit19) [inline app/src/main/res/layout/item_ticket.xml]: add a blank line

File: app/src/main/res/layout/item_ticket.xml [hunk 3]
```diff
@@ -30,23 +30,27 @@
             android:layout_weight="1"
             android:textSize="@dimen/text_size_large"
             tools:text="$25" />
-
```
Comment (nikit19) [inline app/src/main/res/layout/item_ticket.xml]: should be a blank line here IMO

**Comment by simarsingh24** [comment]
@nikit19 @dreadpool2 please review

**Pull request merged**